src/styles/ProfilePage.css

```diff
@@ -1,11 +1,6 @@
-body , html {
-  height: 100%;
-}
-
 .button {
```

Comment:
Lembra de Encapsular o CSS.
```suggestion
.Explorer_Page_button {
```

Reply:
Feito